index.html

```diff
@@ -0,0 +1,221 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <script src="https://kit.fontawesome.com/695f3209b8.js" crossorigin="anonymous"></script>
+    <title>Portfolio Desktop</title>
+</head>
+
+<body>
+    <header>
+        <nav>
+            <div class="nav-left">
+                <span>Atilla Taha</span>
+            </div>
+            <div class="nav-right">
+                <ul>
+                    <li><a href="#">Portfolio</a></li>
+                    <li> <a href="#">About</a> </li>
+                    <li> <a href="#">Contact</a> </li>
+                    <li class="fas fa-envelope-square"></li>
+                </ul>
+            </div>
+        </nav>
+    </header>
+    <div class="main">
+        <section class="intro-page">
+            <div class="intro-social-icon">
+                <ul>
+                    <li><a href="https://twitter.com/AtillaTahaa" class="fa fa-twitter fa-2x icon"></a></li>
+                    <li><a href="#" class="fa fa-medium icon fa-2x"></a></li>
+                    <li><a href="https://github.com/AtillaTahak" class="fa fa-github-square fa-2x icon"></a></li>
+                    <li><a href="https://www.linkedin.com/in/atilla-taha-kördüğüm-a93702186/"
+                            class="fa fa-linkedin-square fa-2x icon"></a></li>
+                    <li><a href="#" class="fa fa-angellist fa-2x icon"></a></li>
+                </ul>
+            </div>
+            <div class="intro-text">
+                <h1>Hey There.
+                    Atilla Taha</h1>
+                <h2>I’am a Software Developer</h2>
+                <p>
+                    I can help you build a product , feature or website Look through some of my work and experience! If
+                    you like what you see and have a project you need coded,
+                    don’t hestiate to contact me.
+
+                </p>
+            </div>
```

Comment:
Please add a background image to the Headline section. Kindly see the screenshot below for reference.
![headline](https://user-images.githubusercontent.com/35504149/135470850-823185ba-ea35-4a35-b3f4-854b9fd1bd53.png)